hieradata/production.yaml

```diff
@@ -41,6 +41,9 @@ environment_ip_prefix: '10.3'
 govuk::apps::efg::ssl_certtype: 'sflg'
 govuk::apps::efg::vhost_name: "www.sflg.gov.uk"
 
+govuk::apps::efg::ssl_certtype: 'sflg'
+govuk::apps::efg::vhost_name: "rebuild.sflg.gov.uk"
```

Comment:
I was just doing what I was told [in the story](https://trello.com/c/xkAb7yDA/237-run-a-rebuild-copy-of-the-efg-app) 😉